yarn.lock

```diff
@@ -830,7 +823,7 @@
     "@types/node" "*"
     "@types/tough-cookie" "*"
 
-"@types/retry@*", "@types/retry@^0.12.0":
+"@types/retry@^0.12.0":
```

Comment:
can you try to remove yarn.lock and create it again to see if this dependency disappear?

Reply:
I'm removing `asyn-retry` not just `retry`.
It might be use in some other package?

Reply:
ahhhh, right XD